src/main/webapp/i18n/de/exercise.json

```diff
@@ -93,6 +93,8 @@
             "isText": "Das ist eine Textübung",
             "isFileUpload": "Das ist eine Datei-Upload-Übung",
             "notificiationText": "Optional: Mitteilung über die Änderung",
+            "submissionSuccessful": "Die submission war erfolgreich",
```

Comment:
At the other occurrences, we translated `submission` with `Einreichung`, so we should be consistent here.

Reply:
I adapted the translation